tests/testthat/test-get_vartypes.R

```diff
@@ -1,5 +1,65 @@
 data(proj)
+library(dplyr)
 
-test_that("get_vartypes", {
+test_that("get_vartypes no NAs", {
   expect_true(any(!is.na(get_vartypes(proj)$class)))
 })
+
+test_that("get_vartypes single choice", {
+  sc_var <- proj$metadata |>
+    filter(field_type %in% c("yesno", "dropdown", "radio")) |>
+    pull(field_name)
+  sc_class <- get_vartypes(proj) |> filter(field_name %in% sc_var) |> pull(class)
+  expect_equal(sc_class,rep("integer",length(sc_class)))
+})
+
+test_that("get_vartypes multiple choice", {
+
+  cb_var <- proj$metadata |>
+    filter(field_type == "checkbox") |>
+    pull(field_name)
+
+  if (is.na(proj$project$missing_data_codes)) {
+    cb <- 1
+  } else if (!is.na(proj$project$missing_data_codes)) {
+    cb <- 2
+  }
+  cb_nbr <- proj$metadata |>
+    filter(field_type == "checkbox") |>
+    mutate(n=str_count(select_choices_or_calculations,pattern = '\\|')+cb,1) |>
+    summarise_at("n", sum) |>
```

Comment:
summarise_at has been superseded... probably better to use `summarise(n = sum(n))`

Reply:
done